test/stark_aggregation.sh

```diff
@@ -84,8 +84,8 @@ fi
 
 echo "4. generate the commit polynomicals files  "
 # generate the commit polynomicals files 
-# input files :  $CIRCUIT.c12.wasm  $C12_VERIFIER.zkin.json  $C12_VERIFIER.pil  $C12_VERIFIER.exec
-# output files :  $C12_VERIFIER.cm
+# input files :  $RECURSIVE_CIRCUIT.wasm  $input0/r1_input.zkin.json  $RECURSIVE_CIRCUIT.pil  $RECURSIVE_CIRCUIT.exec
+# output files :  $RECURSIVE_CIRCUIT.cm
```

Comment:
Is it necessary to generate $RECURSIVE_CIRCUIT.cm? If the commit polynomicals files are just to generate $RECURSIVE2_CIRCUIT.circom. Maybe it can be removed.

Reply:
They carry out the operations of gencircom, compile, setup, buildstarkinfo, buildchelpers, buildconstanttree, and verifier_gencircom when processing Recursive1Steps, which is different from C12aSteps where executing c12a_exec to generate c12a.commit. https://github.com/0xPolygonHermez/zkevm-proverjs/blob/main/package.json#L12-L13

Reply:
yes, we have two flags, --norm-stage, and --agg-stage. For stage `witness` to `c12`, we use --agg-stage to indicate the next step that the c12 proof will be only for proof aggregation.  If not (with --norm-stage only), the next step should add `main` component in the circom, and the rootC should be a public input. 

For your question,  the recursive1 is to norm the aggregated proof on field Goldilocks, and recursive2 is to generate the final proof on field bn128.